frontend/src/page/MatchList.re

```diff
@@ -1,12 +1,12 @@
-type match = {
+type _match = {
```

Comment:
This is not directly related to the purpose of the PR. Either the upgrade to BuckleScript 7.x or to graphql_ppx_re caused the type name `match` to be mangled to `match__` by either bsb or the ppx, resulting in `[%graphql ...]` from line 11 complaining about `list(match__)` on line 9. Prefixing the underscore solved this problem for me. If you want me to separate that into a separate commit, fix it in some other way, and/or leave it alone, I will be happy to do it.

Reply:
Errors looked like this

```
  We've found a bug for you!
  /home/antron/codattle/frontend/src/page/MatchList.re 9:17-21    
   7 │
   8 │ type x = {
   9 │   matches: list(match),
  10 │   maxCountOfScripts: int,
  11 │ };

  This type constructor's parameter, `match__`, can't be found. Is it a typo?

  Hint: Did you mean match_?
```